crates/astria-bridge-contracts/src/lib.rs

```diff
@@ -454,13 +454,8 @@ where
                 .expect("must be set if this method is entered"),
         );
 
-        let memo = memo_to_json(&memos::v1::Ics20WithdrawalFromRollup {
-            memo: event.memo.clone(),
-            rollup_block_number,
-            rollup_return_address: event.sender.encode_hex(),
-            rollup_withdrawal_event_id,
-        })
-        .map_err(GetWithdrawalActionsError::encode_memo)?;
+        let memo = memo_to_json(&event, rollup_block_number, rollup_withdrawal_event_id)
```

Comment:
I think we should also put the rollup withdrawal event ID into this constructor (the line `let rollup_withdrawal_event_id = format!("{transaction_hash}.{event_index}");` above.

Reply:
Good point - done in [32b1522](https://github.com/astriaorg/astria/pull/1753/commits/32b15220e4e70a75cc134a150d1c0d3c9fdfd222).